shuffle_sdk/shuffle_sdk.py

```diff
@@ -109,6 +109,12 @@ def base64_encode(a):
     except:
         return base64.b64encode(a).decode()
 
+@shuffle_filters.register
+def shuffle_default(a, default_value):
+    if a in [None, False, 'null', '']:
```

Comment:
Take into account ALL things that could be used in Default. 

Read how "if a:" works. Also 'none' and "None" as strings were relevant values. This doesn't take into account a lot of data types.

Also, spacing. make it actually readable by adding a newline after the if-statement.